tests/check_case_conflict_test.py

```diff
@@ -46,6 +73,18 @@ def test_file_conflicts_with_committed_file(temp_git_dir):
         assert find_conflicting_filenames(['F.py']) == 1
 
 
+def test_file_conflicts_with_committed_dir(temp_git_dir):
+    with temp_git_dir.as_cwd():
+        temp_git_dir.mkdir('dir').join('x').write('foo')
+        cmd_output('git', 'add', '-A')
+        cmd_output('git', 'commit', '--no-gpg-sign', '-n', '-m', 'Add f.py')
+
+        temp_git_dir.join('DIR').write('foo')
```

Comment:
this is failing on windows -- I don't think you need to create the actual filesystem here (?)

Reply:
the `check_case_conflict` use `git ls-files` to list all files in a repo, so I don't see any way that emulate this without touching a filesystem. My first think was to disable these tests on windows, as with that check we want to prevent this from happening on systems, which are case sensitive

Reply:
hmmm yeah maybe that's fine -- could also emulate the output using some mocking tricks, up to you though